packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/lib/create_saved_objects.ts

```diff
@@ -92,20 +132,24 @@ export const createSavedObjects = async <T>({
   const resolvableErrors = ['conflict', 'ambiguous_conflict', 'missing_references'];
   let expectedResults = objectsToCreate;
   if (!accumulatedErrors.some(({ error: { type } }) => resolvableErrors.includes(type))) {
-    const bulkCreateResponse = await savedObjectsClient.bulkCreate(objectsToCreate, {
-      namespace,
-      overwrite,
-      refresh,
-    });
+    const bulkCreateResponse = await savedObjectsClient.bulkCreate(
+      [...objectsToCreate, ...Array.from(legacyAliases.values())],
+      { namespace, overwrite, refresh }
+    );
```

Comment:
As discussed sync: Ideally, to avoid impact on edge cases, we would create the objects first, and then the legacy url aliases. But that's fine.

Reply:
My bad, for some reason I though we agreed on the opposite :slightly_smiling_face: Anyway, I'll separate these calls just to be on the safe side.

Reply:
My understanding was that we agreed that having a single bulkCreate was totally acceptable as it was easier, compared to do 2 distinct calls 😅

Reply:
Right, right, I've just "heard" what I wanted to hear :slightly_smiling_face: